### PR TITLE
Prepare Freddy for publication on hex.pm

### PR DESCRIPTION
**Pull request by take-five**
* Update package information
* Fix documentation syntax
* Fix typos

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -3,12 +3,13 @@ defmodule Freddy.Mixfile do
 
   def project do
     [app: :freddy,
-     version: "1.0.0",
+     version: "0.9.0",
```
Comment (take-five) [inline mix.exs]: 1.0 will be released when Freddy.RPC.Server is implemented )

File: mix.exs [hunk 2]
```diff
@@ -41,7 +42,14 @@ defmodule Freddy.Mixfile do
       {:poison, ">= 2.0.0"},
 
       {:mock, "~> 0.2.0", only: :test},
+      {:ex_doc, ">= 0.0.0", only: :dev},
       {:dialyxir, "~> 0.5", only: :dev, runtime: false}
     ]
   end
+
+  defp package do
+    [maintainers: ["SaleMove team"],
```
Comment (willmore) [inline mix.exs]: "SaleMove TechMovers"

Comment (take-five) [inline mix.exs]: "TechMovers" doesn't really make any sense to an outside world

Comment (willmore) [inline mix.exs]: It will - because we use it often and consistently.

Comment (take-five) [inline mix.exs]: Although I disagree and find using internal company jargon in OO-community inappropriate, f3b8424

**Comment by take-five** [comment]
!merge

**Comment by take-five** [comment]
!check

**Pull request merged**